engines/bladerunner/bladerunner.cpp

```diff
@@ -323,6 +329,20 @@ bool BladeRunnerEngine::startup(bool hasSavegames) {
 	_gameFlags = new GameFlags();
 	_gameFlags->setFlagCount(_gameInfo->getFlagCount());
 
+#if BLADERUNNER_RESTORED_CONTENT_GAME
+	// EDS flags
+	_extraGameFlagsForRestoredContent = new GameFlags(); //aux flags - custom
```

Comment:
I think this is an overkill. You could just have it a normal GF_ and then set it at the presence of the subtitles file.

Reply:
Then I suggest switching to the normal GF_. This will simplify the code here.

Reply:
This was changed entirely and integration with ScummVM configuration manager has been implemented.